mlflow/environment_variables.py

```diff
@@ -768,3 +768,18 @@ def get(self):
 #: this environment variable directly.
 #: (default: ``None``)
 _MLFLOW_ACTIVE_MODEL_ID = _EnvironmentVariable("_MLFLOW_ACTIVE_MODEL_ID", str, None)
+
+#: Maximum number of parameters to include in the initial CreateLoggedModel request.
+#: Additional parameters will be logged in separate requests.
+#: (default: ``100``)
+MLFLOW_CREATE_LOGGED_MODEL_PARAMS_BATCH_SIZE = _EnvironmentVariable(
```

Comment:
@artjen Can we avoid documenting / exposing this to customers initially? It's good to have an override in an emergency (e.g. critical user escalates a problem), but I hope that customers never have to configure / reason about this variable.

All of the public env vars in this file are documented by default.

```suggestion
_MLFLOW_CREATE_LOGGED_MODEL_PARAMS_BATCH_SIZE = _EnvironmentVariable(
```

Reply:
cc @raymondzhou-db